commit 26498394be2ac122cd3f6e282810410adfb03775 (refs/changes/51/3451/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-02-23T00:38:18+00:00 (1 year, 7 months ago)
    
    TRITON-176 Move FWAPI to new Triton origin image
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/Makefile b/Makefile
index ab2c976..a151103 100644
--- a/Makefile
+++ b/Makefile
@@ -42,10 +42,10 @@ SMF_MANIFESTS_IN = smf/manifests/fwapi.xml.in
 
 include ./tools/mk/Makefile.defs
 ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_VERSION=v0.10.26
+	# Allow building on a SmartOS image other than sdc-*-multiarch 15.4.1.
+	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
+	NODE_PREBUILT_VERSION=v0.10.48
 	NODE_PREBUILT_TAG=zone
-	# Allow building on a SmartOS image other than sdc-smartos/1.6.3.
-	NODE_PREBUILT_IMAGE=fd2cc906-8938-11e3-beab-4359c665ac99
 	include ./tools/mk/Makefile.node_prebuilt.defs
 else
 	NPM_EXEC :=
